Makefile

```diff
@@ -12,12 +12,12 @@ LDFLAGS = -w -X $(PKG)/pkg.Version=$(VERSION) -X $(PKG)/pkg.Commit=$(COMMIT)
 XBUILD = CGO_ENABLED=0 go build -a -tags netgo -ldflags '$(LDFLAGS)'
 BINDIR = bin/mixins/$(MIXIN)
 
-CLIENT_PLATFORM = $(shell go env GOOS)
-CLIENT_ARCH = $(shell go env GOARCH)
+CLIENT_PLATFORM ?= $(shell go env GOOS)
+CLIENT_ARCH ?= $(shell go env GOARCH)
 RUNTIME_PLATFORM ?= linux
```

Comment:
Are `RUNTIME_PLATFORM` and `RUNTIME_ARCH` still relevant/needed?  (Yes, if the `build-runtime` target is still needed, right?)

Reply:
Yeah the RUNTIME_PLATFORM is still needed for when you are building locally, not for a publish. xbuild-all is for when we want to do the entire build matrix. build-client and build-runtime is just enough to build the 2 binaries necessary to get your bin populated with what you need to use porter from source.

These are variables because eventually the runtime os/arch will be configurable (for example arm or windows).